src/config/constants/farms.ts

```diff
@@ -12,6 +12,1129 @@ const farms: FarmConfig[] = [
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 208,
+    lpSymbol: 'NULS-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xaB46737CAAFbB99999f8b91E4D3C6D4D28E10e05',
+    },
+    token: tokens.nuls,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 207,
+    lpSymbol: 'BELT-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x446ff2C0F5350bF2dadD0e0F1AaAA573b362CA6B',
+    },
+    token: tokens.belt,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 206,
+    lpSymbol: 'RAMP-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x6ED589e69D1927AC45054cBb6E57877879384d6F',
+    },
+    token: tokens.ramp,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 205,
+    lpSymbol: 'BFI-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xFFA2357f1E6f48d74b1c18c363c3Fe58A032405a',
+    },
+    token: tokens.bfi,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 204,
+    lpSymbol: 'DEXE-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x534b0b0700c0cfF9785852707f07f60E7C0bc07E',
+    },
+    token: tokens.dexe,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 203,
+    lpSymbol: 'BEL-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x2013265224E3cB6A53C67130F9Fe53Ae36CFcfdd',
+    },
+    token: tokens.bel,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 202,
+    lpSymbol: 'TPT-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0xC14c2dd21d6aEA3C2068A1F8e58d41D3c28F9288',
+    },
+    token: tokens.tpt,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 201,
+    lpSymbol: 'WATCH-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xD5fBfFf5faB9d29f614d9bd50AF9b1356C53049C',
+    },
+    token: tokens.watch,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 200,
+    lpSymbol: 'xMARK-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x601aE41C5a65b2089a6af2CcfaF984896a1f52AD',
+    },
+    token: tokens.xmark,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 199,
+    lpSymbol: 'bMXX-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0x037d38c7DfF5732DAA5f8C05478Eb75cdf24f42B',
+    },
+    token: tokens.bmxx,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 198,
+    lpSymbol: 'IOTX-BUSD LP',
+    lpAddresses: {
+      97: '',
+      56: '0x8503462D0d4D3ce73e857bCC7D0Ef1125B0d66fF',
+    },
+    token: tokens.iotx,
+    quoteToken: tokens.busd,
+  },
+  {
+    pid: 197,
+    lpSymbol: 'BOR-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xE0243Ce3b50bd551168cE6964F178507d0a1acD5',
+    },
+    token: tokens.bor,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 196,
+    lpSymbol: 'bOPEN-BNB LP',
+    lpAddresses: {
+      97: '',
+      56: '0xD2FcF98EaeD2c08e9BcA854802C07b93D27913aC',
+    },
+    token: tokens.bopen,
+    quoteToken: tokens.wbnb,
+  },
+  {
+    pid: 195,
+    lpSymbol: 'SUSHI-ETH LP',
+    lpAddresses: {
+      97: '',
+      56: '0x3BECbb09F622187B544C0892EeDeB58C004117e1',
+    },
+    token: tokens.sushi,
+    quoteToken: tokens.eth,
+  },
+  {
```

Comment:
Anyone checking - 193 missing because it's the SWINGBY farm, highlighted near the top of the file